packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -312,14 +321,12 @@ export class AzureCommunicationCallWithChatAdapter implements CallWithChatAdapte
   }
   /** Dispose of the current CallWithChatAdapter. */
   public async dispose(): Promise<void> {
-    this.chatAdapterPromise.then((adapter) => {
+    this.executeWithResolvedChatAdapter((adapter) => {
```

Comment:
This function should not throw an error if the chat adapter is not initialized yet, its a valid scenario to dispose of the adapter before the chat is initialized. If this happens any chatAdapter promises lying around need cleaned up also.